css-layout-classes/styles.css

```diff
@@ -0,0 +1,31 @@
+.container {
+  max-width: 800px;
+  margin: auto;
+
+  /* margin-right: auto;
+  margin-left: auto; */
+}
+
+.row {
+  display: flex;
+}
+
+.column-three {
+  flex-basis: calc(100% / 3);
+}
+
+.column-four {
+  flex-basis: calc(100% / 4);
+}
+
+.column-two-third {
+  width: 66%;
+}
+
+.column-one-third {
+  width: 33%;
+}
```

Comment:
Oh, you already have a class to make one third, you can remove the above class, and just reuse this one.